src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -382,26 +386,79 @@ function WorkspaceMembersPage({policyMembers, personalDetails, route, policy, se
         </View>
     );
 
+    const changeUserRole = (role: typeof CONST.POLICY.ROLE.ADMIN | typeof CONST.POLICY.ROLE.USER) => {
+        if (!isEmptyObject(errors)) {
+            return;
+        }
+
+        const accountIDsToUpdate = selectedEmployees.filter((id) => policyMembers?.[id].role !== role);
+
+        Policy.updateWorkspaceMembersRole(route.params.policyID, accountIDsToUpdate, role);
+        setSelectedEmployees([]);
+    };
+
+    const getBulkActionsButtonOptions = () => {
+        const iconSettings = {
+            iconWidth: variables.iconSizeExtraLarge,
+            iconHeight: variables.iconSizeExtraLarge,
+        };
+        const options: Array<DropdownOption<WorkspaceMemberBulkActionType>> = [
+            {
+                text: translate('workspace.people.removeMembersTitle'),
+                value: CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.REMOVE,
+                icon: Expensicons.RemoveMembers,
+                onSelected: askForConfirmationToRemove,
+                ...iconSettings,
+            },
+        ];
+
+        if (selectedEmployees.find((employee) => policyMembers?.[employee].role === CONST.POLICY.ROLE.ADMIN)) {
+            options.push({
+                text: translate('workspace.people.makeMember'),
+                value: CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_MEMBER,
+                icon: Expensicons.MakeMember,
+                onSelected: () => changeUserRole(CONST.POLICY.ROLE.USER),
+                ...iconSettings,
+            });
+        }
+        if (selectedEmployees.find((employee) => policyMembers?.[employee].role === CONST.POLICY.ROLE.USER)) {
```

Comment:
NAB just an extra line for readability
```suggestion

        if (selectedEmployees.find((employee) => policyMembers?.[employee].role === CONST.POLICY.ROLE.USER)) {
```

Reply:
added